Cargo.toml

```diff
@@ -156,6 +156,7 @@ zip = { version = "0.6.6", default-features = false, features = ["zstd"] }
 [workspace.lints.rust]
 unsafe_code = "warn"
 unreachable_pub = "warn"
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(fuzzing)'] }
```

Comment:
Nice, this will make the fuzzer build output less noisy as for MacOS it always required to use the nightly compiler